src/cbc_sdk/platform/devices.py

```diff
@@ -414,6 +415,54 @@ def remove_from_groups(self, groups):
         if len(actual_groups) > 0:
             self._refresh()
 
+    def preview_add_policy_override(self, policy_id):
```

Comment:
I'm a bit confused by this as a policy_override for a device would always result in the policy being applied so what are we actually previewing here?

Reply:
Remove policy override makes sense as we want to know what will be assigned but adding an override seems redundant

Reply:
If you think it's not needed, I'll just ditch that particular helper.  It can be erased easily without affecting anything else.

Reply:
Yeah it doesn't seem useful as it will only show the change you are making and won't have any unknown discoverable impacts

Reply:
Removing it, and patching the appropriate unit test to point to `Device.preview_add_policy_override_for_devices`.